ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -8,22 +8,27 @@ import UIKit
 
 class DemoListViewController: UITableViewController {
 
-    static func addDemoListTo(window: UIWindow, pushing viewController: UIViewController?) {
-        if let colorProvider = window as? ColorProviding, let primaryColor = colorProvider.primaryColor(for: window) {
-            Colors.setProvider(provider: colorProvider, for: window)
-            FluentUIFramework.initializeAppearance(with: primaryColor, whenContainedInInstancesOf: [type(of: window)])
-        } else {
-            FluentUIFramework.initializeAppearance(with: Colors.primary(for: window))
-        }
+    private var provider: ColorProviding = DemoColorDefaultTheme() as ColorProviding
```

Comment:
This cast shouldn't be necessary; the assignment should automatically pick up that `DemoColorDefaultTheme()` is a `ColorProvider`.